src/magellan_vision/scripts/detect_targets.py

```diff
@@ -126,20 +133,25 @@ def detectArch(self):
             return (None, None)
 
     def detectRamp(self):
-        # frame = cv.imread('Ramp.png', cv.IMREAD_COLOR)
         frame = self.frame
         frameHSV = cv.cvtColor(frame, cv.COLOR_BGR2HSV)
         rampThreshold = cv.inRange(frameHSV, (self.hueMinRamp, self.satMin, self.valMin),
                                    (self.hueMaxRamp, self.satMax, self.valMax))
-        center = ndimage.measurements.center_of_mass(rampThreshold)
-        centerImage = cv.circle(frame, (int(center[1]), int(center[0])), 10, (0, 0, 255), 3)  # noqa: F841
-        if self.debugOutput:
-            # cv.imshow("threshold", centerImage)
-            cv.waitKey(0)
+        rampSize = np.sum(np.sum(rampThreshold))
+        imgSize = np.size(rampThreshold)
+        # Don't detect a ramp if the area is less than 1% of the image
+        if rampSize < (imgSize / 100):
+            center = None
+        else:
+            center = ndimage.measurements.center_of_mass(rampThreshold)
```

Comment:
You could make this one line by doing `center = ndimage.measurements.center_of_mass(rampThreshold) if else None`

Then below do `if center and self.debugOutput:`